### DISCUSSION TITLE
Asking specific email

### DISCUSSION THREAD

**Discussion by bernardocasanova** [community/community#32324] [Copilot Conversations]
When i try get my license version of copilot for student they asking me a specific email from my school, but i have NOT this email.. i have my card for students, can i use this?

**Comment by iamcjwilliams** [comment]
Hello @bernardocasanova We're sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)
